src/cmd/tools/archiver.go

```diff
@@ -42,6 +48,27 @@ var archiverDecompressCmd = &cobra.Command{
 		if err != nil {
 			message.Fatal(err, lang.CmdToolsArchiverDecompressErr)
 		}
+
+		// Decompress component layers in the destination path
+		if decompressLayers {
```

Comment:
TODO: the entire `decompress` change needs to be re-evaluated. If we are still doing it, this code needs to be changed as it does not use viper.